flutter/shell/platform/tizen/nui_autofill_popup.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/nui_autofill_popup.h"
+
+#include <dali-toolkit/dali-toolkit.h>
+#include <dali-toolkit/devel-api/controls/table-view/table-view.h>
+#include <dali-toolkit/public-api/controls/text-controls/text-label.h>
+
+#include "flutter/shell/platform/tizen/tizen_autofill.h"
+
+namespace flutter {
+
+bool NuiAutofillPopup::Touched(Dali::Actor actor,
+                               const Dali::TouchEvent& event) {
+  const Dali::PointState::Type state = event.GetState(0);
+  if (Dali::PointState::DOWN == state) {
+    std::string text =
+        actor.GetProperty(Dali::Actor::Property::NAME).Get<std::string>();
+    on_commit_(text);
+    popup_.SetDisplayState(Dali::Toolkit::Popup::HIDDEN);
+  }
+  return true;
+}
+
+void NuiAutofillPopup::Hidden() {
+  // TODO(Swanseo0): There is a phenomenon where white traces remain for a
+  // while when popup disappears.
+  popup_.Unparent();
+  popup_.Reset();
+}
+
+void NuiAutofillPopup::OutsideTouched() {
+  popup_.SetDisplayState(Dali::Toolkit::Popup::HIDDEN);
+}
+
+void NuiAutofillPopup::Prepare() {
+  popup_ = Dali::Toolkit::Popup::New();
+  popup_.SetProperty(Dali::Actor::Property::NAME, "popup");
+  popup_.SetProperty(Dali::Actor::Property::PARENT_ORIGIN,
+                     Dali::ParentOrigin::TOP_LEFT);
+  popup_.SetProperty(Dali::Actor::Property::ANCHOR_POINT,
+                     Dali::AnchorPoint::TOP_LEFT);
+  popup_.SetProperty(Dali::Toolkit::Popup::Property::TAIL_VISIBILITY, false);
+  popup_.SetBackgroundColor(Dali::Color::WHITE_SMOKE);
+  popup_.OutsideTouchedSignal().Connect(this,
+                                        &NuiAutofillPopup::OutsideTouched);
+  popup_.HiddenSignal().Connect(this, &NuiAutofillPopup::Hidden);
+  popup_.SetProperty(Dali::Toolkit::Popup::Property::BACKING_ENABLED, false);
+  popup_.SetProperty(Dali::Toolkit::Popup::Property::AUTO_HIDE_DELAY, 2500);
```

Comment:
Is AUTO_HIDE_DELAY an intended property?
If a value is set, it will automatically hide after that amount of time.
I think, this value looks a property used for toast popup.
If this is the intended concept, please ignore this comment.

Reply:
This Property is intended.
At first, the situation when pop-up was hidden was as follows.
- Select an item
- Press Back
- Touch outside

However, when `popup_.SetProperty(Dali::Toolkit::Popup::Property::BACKING_ENABLED, false)` was added, `OutsideTouched` was not executed. So I added `AUTO_HIDE_DELAY`.